solana-programs/reward-manager/cli/src/main.rs

```diff
@@ -257,6 +258,41 @@ fn command_delete_sender_public(
     transaction.sign(config, 0)
 }
 
+fn command_change_reward_manager_authority(
+    config: &Config,
+    reward_manager: Pubkey,
+    current_authority: Pubkey,
+    new_authority: Pubkey
+) -> CommandResult {
+    println!("Using reward manager - {:}", &reward_manager);
+    println!("Current authority input - {:}", &current_authority);
+    println!("New authority - {:}", &new_authority);
+    let reward_manager_data = config.rpc_client.get_account_data(&reward_manager)?;
+    let reward_manager_from_chain = RewardManager::unpack(reward_manager_data.as_slice())?;
+    println!("Current authority from chain - {:}", reward_manager_from_chain.manager);
+
+    if reward_manager_from_chain.manager != current_authority {
+        println!("Incorrect parameters, exiting");
```

Comment:
maybe include the if condition vars as so during debugging mode can figure out what's causing the failure

Reply:
they are printed above!

Reply:
oooo good callout! resolving